inc/Classes/File.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace LanSuite;
+
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Filesystem\Path;
+/**
+ * Class File
+ * Generic class for file accessor functions
+ */
+
+class File
+{
+    private static string $_basePath='';
+    private string $_relativePath='';
+    private Filesystem $_fileSystem;
```

Comment:
I am personally not a fan of the `$_*` way, because the `private` keyword already marks it as a private variable. Any rational behind it?

Reply:
I see a point that it allows to identify private attributes purely on the name without looking up the defintion, but also do agree that access within the code is not as nice as with just camelcase naming.

Reply:
Underscores have been removed